nextcord/application_command.py

```diff
@@ -2968,3 +3105,85 @@ def get_roles_from_interaction(state: ConnectionState, interaction: Interaction)
             ret.append(role)
 
     return ret
+
+
+def unpack_annotated(given_annotation, resolve_list: list = []) -> type:
+    origin = typing.get_origin(given_annotation)
+    if origin is Annotated:
+        located_annotation = MISSING
+        arg_list = typing.get_args(given_annotation)
+        # for i in range(1, len(arg_list)):
+        #     if arg_list[i] in resolve_list:
+        #         located_annotation = arg_list[i]
+        #         break
```

Comment:
Shouldn't this comment block be removed?

Reply:
Should not this comment block be removed?
Perhaps.